examples/cpp/core/polymorphic_container_visitor/polymorphic_container_visitor.bond

```diff
@@ -1,8 +1,15 @@
 namespace examples.polymorphic_containers
 
+enum StructKind
+{
+    StructKind_Unknown;
```

Comment:
Any reason for using `StructKind_` with `c++11`?

Reply:
Bond doesn't yet generate enum classes, so `Circle` would be ambiguous between the enum member and the struct.

Reply:
I thought even old one used to generate in a way that `Enum::Value` works using a `namespace` trick?

Reply:
`Enum::Value` works, but just `Value` is also a visible name. Since--in this case--there is also a struct called `Value` that is visible, just `Value` is ambiguous.

Reply:
Ah, OK.